src/components/ResourceCard/ResourceCardBody/EncounterCardBody.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { shape } from 'prop-types';
+import CardBodyField from './CardBodyField';
+import {
+  getPatientAgeAtResourceDate,
+  getEnding,
+  getClass,
+  getStatus,
+} from '../../../resources/fhirReader';
+import CARD_BODY_LABEL from '../../../resources/cardBodyLabel';
+
+const EncounterCardBody = ({ resource, patientResource }) => {
+  const { type, subType } = resource;
+
+  return (
+    <>
+      <CardBodyField
+        label={CARD_BODY_LABEL.age}
```

Comment:
This at first looks like the condition/vital sign/ etc age, and now "[Patient] Age at measure" as it actually means.